packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -434,6 +435,9 @@ class SemanticsProperties extends DiagnosticableTree {
   ///  * [SemanticsFlag.namesRoute] for a description of how the name is used.
   final bool namesRoute;
 
+  /// If non-null, whether the node should be considered a live region.
```

Comment:
this should explain what that means.

Reply:
Updated comment to point to the config and flag documentation.